packages/tests/config.ts

```diff
@@ -0,0 +1,41 @@
+import dotenv from "dotenv";
+
+type Config = {
+  oid: string;
+  apiKey: string;
+  apiURL: string;
+  app: string;
+  CI: boolean;
+};
+
+dotenv.config();
+
+function createE2ETestConfig() {
+  const config: Config = {
+    oid: process.env.E2E_SID_ORG_ID ?? "",
+    apiKey: process.env.E2E_SID_API_KEY ?? "",
+    apiURL: process.env.E2E_SID_API_URL ?? "",
+    app: process.env.APP_NAME || "",
+    CI: process.env.CI === "true",
+  };
+
+  if (!config.app) {
+    throw new Error("APP_NAME is required in .env file");
+  }
+
+  if (!config.oid) {
+    throw new Error("E2E_SID_ORG_ID is required in .env file");
+  }
+
+  if (!config.apiKey) {
+    throw new Error("E2E_SID_API_KEY is required in .env file");
+  }
+
+  if (!config.apiURL) {
+    throw new Error("E2E_SID_API_URL us required in .env file");
+  }
+
+  return config;
+}
+
+export const config = createE2ETestConfig();
```

Comment:
@kaspermroz sorry, I missed this initially when setting this project up and now in the code review again - there are more env vars in the `email` module that require the same treatment:
```
 const config = {
    serverId: process.env.MAILOSAUR_SERVER_ID || "",
    apiKey: process.env.MAILOSAUR_API_KEY || "",
  };

  if (!config.serverId || !config.apiKey) {
    throw new Error("Mailosaur configuration is missing");
  }
```

Can you add those here as well?

Reply:
Sure! Sorry I forgot about these too 😬 on it